publisher-sdk/src/main/java/com/criteo/publisher/BidResponseListener.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher;
+
+import androidx.annotation.Keep;
+import androidx.annotation.Nullable;
+import com.criteo.publisher.advancednative.CriteoNativeLoader;
+
+@Keep
+public interface BidResponseListener {
+
+  /**
+   * Callback invoked when a response for a bid is given to the publisher.
+   * <p>
+   * In case of no bid, the given response is <code>null</code>. In case of successful bid, the given response is not
+   * <code>null</code> and its CPM is accessible through {@link BidResponse#getPrice()}.
+   *
+   * <h1>InHouse integration</h1>
+   * When using the In-House integration, the response can be provided to one a those methods to display the ads:
+   * <ul>
+   *   <li>{@link CriteoBannerView#loadAd(BidResponse)}: display a banner</li>
+   *   <li>{@link CriteoInterstitial#loadAd(BidResponse)}: display an interstitial</li>
+   *   <li>{@link CriteoNativeLoader#loadAd(BidResponse)}: display a native ad</li>
+   * </ul>
+   * <p>
+   * Please note that you should use the <code>loadAd</code> method corresponding to the kind of ad unit you asked a bid
```

Comment:
Regarding documentation we should be consistent with our tone of voice. Here we are mixing 2nd and 3rd person sentences.

`Callback invoked when a response for a bid is given to the publisher.` -> 3rd person
`Please note that you should use the <code>loadAd</code> method corresponding to the kind of ad unit you asked a bid` -> 2nd person.

Which tone of voice you think is the most appropriate for code documentation?

Reply:
Used 3rd person